internal/gps/maybe_source.go

```diff
@@ -47,7 +46,11 @@ func (mbs maybeSources) try(ctx context.Context, cachedir string, c singleSource
 		if err == nil {
 			return src, state, nil
 		}
-		errs = append(errs, errors.Wrapf(err, "failed to set up %q", mb.getURL()))
+		urls := ""
+		for _, url := range mb.possibleURLs() {
+			urls += url.String() + "\n"
+		}
+		errs = append(errs, errors.Wrapf(err, "failed to set sources from the following URLs:\n%s", urls))
```

Comment:
🤦‍♂️ 

i fatfingered that. i meant `"failed to set up sources from the following URLs:\n"` - sorry!